docs/content/stable/yugabyte-platform/troubleshoot/install-upgrade-issues/installer.md

```diff
@@ -0,0 +1,86 @@
+headerTitle: Install and upgrade issues
+linkTitle: Install and upgrade issues
+description: Troubleshoot issues encountered when installing or upgrading YugabyteDB Anywhere on virtual machines.
+menu:
+  stable_yugabyte-platform:
+    identifier: install-upgrade-installer-issues
+    parent: troubleshoot-yp
+    weight: 10
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li>
+    <a href="../installer/" class="nav-link active">
+      <i class="fa-solid fa-building"></i>
+      YBA Installer</a>
+  </li>
+
+  <li>
+    <a href="../vm/" class="nav-link">
+      <i class="fa-solid fa-cloud"></i>
+      Virtual machine</a>
+  </li>
+
+  <li>
+    <a href="../kubernetes/" class="nav-link">
+      <i class="fa-regular fa-dharmachakra" aria-hidden="true"></i>
+      Kubernetes
+    </a>
+  </li>
+
+</ul>
+
+Occasionally, you might encounter issues during installation and upgrade of YugabyteDB Anywhere using [YBA Installer](../../../install-yugabyte-platform/install-software/installer/).
+
+If you experience difficulties while troubleshooting, contact [Yugabyte Support](https://support.yugabyte.com).
+
+## Permissions issue with find command
+
+YBA Installer preflight check fails with the following error:
+
+```output
+ERROR[2023-12-08T08:25:58Z] Command 'find as user yugabyte' failed with exit code '1' and stderr 'find: Failed to change directory: Permission denied
+find: Failed to change directory: Permission denied
+find: Failed to change directory: Permission denied
+find: Failed to change directory: Permission denied
+find: failed to restore initial working directory: Permission denied
+```
+
+This happens when the user doesn't have sufficient privileges on the `yba_installer_full` directory where the YBA Installer installation package was untarred.
+
+Affected releases: All
+
+Workaround: Change permissions on `yba_installer_full` directory to 755.
+
+## Preflight check for Prometheus fails
+
+Upgrades fail the preflight check with the following error when run as `yba-ctl preflight --upgrade`:
+
+```output
+time=2024-01-03T19:19:04Z level=error msg=preflight prometheus failed: couldn't parse either scrapeInterval: 15 or scrapeTimeout: 10 to duration. check https://pkg.go.dev/time#ParseDuration for appropriate syntax
+```
+
+Affected releases: Upgrading from YBA versions earlier than v2.18.5 to v2.18.5 and later using YBA Installer.
+
+Workaround: Ignore this preflight check. YBA Installer automatically converts the invalid settings to valid values and performs the upgrade.
+
+If you want to have green preflight checks prior to the upgrade, change the `scrapeInterval` and `scrapeTimeout` settings in `/opt/yba-ctl/yba-ctl.yml` to `10s` and `15s` respectively.
+
+## New settings are not automatically updated in yba-ctl.yml
+
+When new default settings are added to `yba-ctl.yml` in a newer release, they aren't automatically added to `yba-ctl.yml` of an existing installation after an upgrade.
```

Comment:
Change aren't to "may not" because we do have migrations for some settings, but we forgot a couple.